web/src/components/error-page.tsx

```diff
@@ -24,3 +26,21 @@ export const ErrorPage = ({
     </div>
   );
 };
+
+export const ErrorPageWithSentry = ({
+  title = "Error",
+  message,
+}: {
+  title?: string;
+  message: string;
+}) => {
+  useEffect(() => {
+    // Capture the error with Sentry
+    if (window !== undefined)
+      captureException(
```

Comment:
Should this only be triggered if we have the sentry env variable available?

Reply:
this "noops" when no sentry env is set

related issue: https://arc.net/l/quote/lcagvjcp

Reply:
tested, works without sentry env without further wrapping